.eslintrc.js

```diff
@@ -27,6 +30,9 @@ module.exports = {
         "semi": [
             "error",
             "always"
+        ],
+        "space-in-brackets": [
```

Comment:
The linter currently fails on this rule; please fix this somehow prior to merge.